lib/mayaUsd/fileio/registryHelper.cpp

```diff
@@ -254,7 +254,7 @@ VtDictionary UsdMaya_RegistryHelper::GetComposedInfoDictionary(const std::vector
     }
 
     // Validate that keys are only defined once globally.
-    for (const std::pair<std::string, std::vector<std::string>>& pair : keyDefinitionSites) {
+    for (const std::pair<std::string, std::vector<std::string>> pair : keyDefinitionSites) {
```

Comment:
please use auto. 
`for (const auto pair : keyDefinitionSites)`

Once we switch to C++17, We can leverage the  Structured bindings feature.